composer.json

```
@@ -10,6 +10,7 @@
         }
     ],
     "require": {
+        "php": "^5.6 || ^7.0",
```

```suggestion
        "php": "^7.1",
```

and also `ezsystems/ezplatform-admin-ui` should be added to dependencies  // cc @lserwatka

---

Yes, and could you change branch alias to `1.0.x-dev` this package is super stable.

---

not for 0.2.x, for followup where we change admin ui to allow extensibility on system info tab and add eZ tab we can do things like this

This  is 0.2.x for reasons explained in description, TL;DR to make all 2.x users aware if they are on a unsupported version.